src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -61,15 +83,15 @@ public void doGet(
   /**
    * Get the names of the tasks in all of the user's tasklists
    *
-   * @param tasksService a valid tasksService with a valid credential
+   * @param tasksClient either a mock TaskClient or a taskClient with a valid credential
    * @return List of tasks from user's account
    * @throws IOException if an issue occurs with the tasksService
```

Comment:
nit: change `tasksService` to `tasksClient`

Reply:
It currently is tasksClient